api/src/main/java/org/phenotips/mendelianSearch/internal/MendelianVariantCategory.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/
+ */
+package org.phenotips.mendelianSearch.internal;
+
+import org.xwiki.component.annotation.Component;
+
+import java.util.List;
+
+/**
+ * TODO improve this javadoc
+ *
+ * A MendelianVariantCategory object describes a single category of variants that
+ * may be used to filter searches in the Mendelian Search Application.
+ * A variant category contains multiple variant effects that are used to query the
+ * variant store.
+ *
+ * @version $Id$
+ * @since 1.0
+ */
+@Component
```

Comment:
This annotation only makes sense if it's supposed to be an injectable component implementation, in which case it must also have some other class or interface annotated with `@Role` in its hierarchy. Just remove this annotation.